script_library/deployment-actions-kvm.sh

```diff
@@ -63,39 +36,6 @@ function remove_compute(){
     echo "Now Remove compute"
     run_ansible ${socok8s_absolute_dir}/playbooks/remove_compute.yml -e compute_node_name=$1
 }
-function deploy_tempest(){
-    echo "Deploying Tempest"
-    run_ansible ${socok8s_absolute_dir}/playbooks/deploy_tempest.yml
-}
-function clean_k8s(){
-    echo "DANGER ZONE. Set the env var 'DELETE_ANYWAY' to 'YES' to delete everything in your userspace."
-    if [[ ${DELETE_ANYWAY:-"NO"} == "YES" ]]; then
-        run_ansible ${socok8s_absolute_dir}/playbooks/generic-clean_k8s.yml
-    fi
-}
-function clean_airship(){
-    clean_action=''
-    action_desc='everything'
-    if [[ "${1:-default}" != 'default' ]]; then
-        clean_action=" -e clean_action=$1"
-        action_desc=$1
-    fi
-    echo "DANGER ZONE. Set the env var 'DELETE_ANYWAY' to 'YES' to delete airship artifacts ( ${action_desc} ) in your userspace."
-    if [[ ${DELETE_ANYWAY:-"NO"} == "YES" ]]; then
-        run_ansible ${socok8s_absolute_dir}/playbooks/generic-clean_airship.yml ${clean_action}
-    fi
-}
 function clean_kvm(){
     echo "Not implemented"
 }
-function clean_userfiles(){
-    echo "DANGER ZONE. Set the env var 'DELETE_ANYWAY' to delete everything in your userspace."
-    if [[ ${DELETE_ANYWAY:-"NO"} == "YES" ]]; then
-        extra_arg="-e delete_anyway='yes'"
-    fi
-    run_ansible ${socok8s_absolute_dir}/playbooks/generic-clean_userfiles.yml ${extra_arg:-}
-}
-function teardown(){
```

Comment:
This is different to the openstack or common implementations.

Reply:
Yes, but I do not think adding `gather_logs` option into common part hurts.

Reply:
No, but this is doing clean_kvm, and the new function does clean_openstack

Reply:
Ah, true, I missed that.